### ISSUE TITLE
Add supported graphics section in docs

### ISSUE THREAD

**Issue by dmey** [openjournals/joss#473]
I have noticed that Scalable Vector Graphics (SVG) figures are not supported. Given that `pdflatex` is used to generate the pdf, I think the only supported formats are `pdf`, `png`, `jpg`, and`eps`. I think it would help to have a supported media-formats section. Happy to send a PR for the docs to include a table with supported formats.

**Comment by jedbrown** [comment]
I haven't tested it with a paper through whedon, but Pandoc supports SVG by using `rsvg-convert`. https://github.com/jgm/pandoc/commit/6b72c5e35ba9b0872704b77b2bd8793657adcaba
Do you have an example that fails?

@arfon Is `rsvg-convert` (`librsvg`) installed on Whedon's host?

**Comment by dmey** [comment]
@jedbrown https://github.com/openjournals/joss-reviews/issues/1113#issuecomment-443378686

**Comment by jedbrown** [comment]
Aha, I ran it locally and it worked, though `rsvg-convert` botched the text wrapping so the diagram doesn't look good. I haven't run into this before and it would be worth [filing a bug report](https://gitlab.gnome.org/GNOME/librsvg/issues).

@arfon I think it's worth installing `librsvg` -- it's a widely used tool and usually produces good output.

**Comment by arfon** [comment]
@jedbrown - it looks like `librsvg2` is already installed on the Heroku hosts: https://devcenter.heroku.com/articles/stack-packages

> Aha, I ran it locally and it worked, though rsvg-convert botched the text wrapping so the diagram doesn't look good. I haven't run into this before and it would be worth filing a bug report.

Does anything specific have to be passed to Pandoc or should it 'just work'?

**Comment by jedbrown** [comment]
It just works when I run locally. Is the executable rsvg-convert in PATH. That's what the implementation tries.On Dec 15, 2018 04:14, Arfon Smith <notifications@github.com> wrote:@jedbrown - it looks like librsvg2 is already installed on the Heroku hosts: https://devcenter.heroku.com/articles/stack-packages

Aha, I ran it locally and it worked, though rsvg-convert botched the text wrapping so the diagram doesn't look good. I haven't run into this before and it would be worth filing a bug report.

Does anything specific have to be passed to Pandoc or should it 'just work'?

**Comment by jedbrown** [comment]
Looks like you need to install [librsvg2-bin](https://packages.ubuntu.com/cosmic/amd64/librsvg2-bin/filelist).

**Comment by letmaik** [comment]
What's the status for SVG support? I just tried to use https://github.com/openbases/openbases-pdf and got "rsvg-convert: createProcess: runInteractiveProcess: exec: does not exist (No such file or directory)".

**Comment by arfon** [comment]
I've not had time to get this working sorry. Please use another format (PNG etc.)

**Comment by jedbrown** [comment]
A `pdf` figure will look better (as a vector graphic).

**Comment by danielskatz** [comment]
@arfon - Do we now accept SVG figures?  If so, can you close this issue?

**Comment by arfon** [comment]
Yes, I believe we do. It broke at one point but was fixed in https://github.com/openjournals/inara/issues/65
